openvpn/client_config.go

```diff
@@ -26,6 +27,13 @@ type ClientConfig struct {
 	*config.GenericConfig
 }
 
+// SetConnectOptions sets connect options provided through endpoint parameters
+func (c *ClientConfig) SetConnectOptions(options dto.ConnectOptions) {
```

Comment:
Client config should have clear methods -> RestrictReconnects and so on, dont add external dependency as ConnectOptions